src/components/app.scss

```diff
@@ -17,6 +17,11 @@
     top: $topBarHeight;
     left: 0;
     transition: $rightPanelTransition;
+
+    > div:nth-child(2) {//override 3js default 100% width to center terrain with sim info div on left
```

Comment:
I think this is a very fragile selector. It'd be better to add explicit className if it's not defined yet.